features/utils/credentials.ts

```diff
@@ -206,6 +211,7 @@ const loadFromKintone: () => Promise<Credentials> = async () => {
       (r): AppCredential => ({
         key: r.key.value,
         appId: r.app_id.value,
+        guestSpaceId: r.guest_space_id.value,
```

Comment:
@hung-cybo 
Thanks for implementing. I have one considering.
Is it guestSpaceId suitable in appCredentials context?
Do we have any place to get and set this value instead of credential context.
Thanks for making clear.

Reply:
@tuanphamcybozu 

I have replied on the thread.
You can check it now.
Thank you.